helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -484,6 +484,8 @@ processors:
     filter:
       node_from_env_var: K8S_NODE_NAME
 
+  {{- include "splunk-otel-collector.k8sAttributesProcessorMetrics" . | nindent 2 }}
```

Comment:
I think this should be limited to pulling pod data local to the agent node only. So adding the config to this processor
```
    filter:
      node_from_env_var: K8S_NODE_NAME
```